libs/features/personalization/personalization.js

```diff
@@ -180,16 +180,15 @@ function normalizeKeys(obj) {
 function handleCommands(commands, manifestId, rootEl = document) {
   commands.forEach((cmd) => {
     if (VALID_COMMANDS.includes(cmd.action)) {
-      try {
-        let selectorEl = rootEl.querySelector(cmd.selector);
-        if (!selectorEl) return;
-        if (selectorEl.classList[0] === 'section-metadata') {
-          selectorEl = selectorEl.parentElement || selectorEl;
-        }
-        COMMANDS[cmd.action](selectorEl, cmd.target, manifestId);
-      } catch (e) {
-        console.log('Invalid selector: ', cmd.selector);
+      let selectorEl = rootEl.querySelector(cmd.selector);
```

Comment:
I think the `try/catch` block might've made sense here, since I see that unit tests are failing
> SyntaxError: Failed to execute 'querySelector' on 'Document': ''/.marqueee' is not a valid selector.

The actual failing test is https://github.com/adobecom/milo/blob/main/test/features/personalization/personalization.test.js#L145

Reply:
right, we need to catch an error in case someone accidentally will use an invalid selector in MEP manifest.
Please see [https://github.com/adobecom/milo/pull/1348](https://github.com/adobecom/milo/pull/1348)